pkgs/applications/misc/ddcui/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, wrapQtAppsHook
+, qtbase
+, qttools
+, ddcutil
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ddcui";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "rockowitz";
+    repo = "ddcui";
+    rev = "v${version}";
+    sha256 = "02qr7i3pdq8p6lnhwihfgd9b7y9zwpdk6gwv626rz32ai6mfrfhl";
+  };
+
+  nativeBuildInputs = [
+    # Using cmake instead of the also-supported qmake because ddcui's qmake
+    # file is not currently written to support PREFIX installations.
+    cmake
+    pkg-config
+    wrapQtAppsHook
```

Comment:
You can remove wrapQtAppsHook and qtbase from inputs using proper Qt wrapping, see https://github.com/NixOS/nixpkgs/issues/65399.

Reply:
> You can remove [...] qtbase

That's a bit surprising, given that the manual gives it as the main example:

https://hydra.nixos.org/build/96804884/download/1/nixpkgs/manual.html#sec-language-qt

So is it now recommended to include it, or not?

Reply:
Well, I don't know why it is recommended to include in the manual. I just see a lot of packages without explicit qtbase input.

Reply:
OK, then I'll stick with including it for now to be as close to the manual as possible.